keanu-python/keanu/sequence.py

```diff
@@ -108,11 +109,32 @@ def get_last_item(self) -> SequenceItem:
         return SequenceItem(self.unwrap().getLastItem())
 
     @staticmethod
-    def proxy_for(label: str) -> str:
+    def proxy_label_for(label: str) -> str:
         """
-        >>> Sequence.proxy_for("foo")
+        >>> Sequence.proxy_label_for("foo")
         'proxy_for.foo'
         """
         label_java = _VertexLabel(label).unwrap()
-        proxy_label_java = k.jvm_view().SequenceBuilder.proxyFor(label_java)
+        proxy_label_java = k.jvm_view().SequenceBuilder.proxyLabelFor(label_java)
         return proxy_label_java.getQualifiedName()
+
+    @staticmethod
+    def double_proxy_for(label: str, shape: Collection[int] = ()) -> Vertex:
+        """
+        Creates a proxy vertex for the given label
+        """
+        return DoubleProxy(shape, Sequence.proxy_label_for(label))
```

Comment:
I'd prefer you to call `k.jvm_view().SequenceBuilder.doubleProxyFor` here. Otherwise you are duplicating logic in java and python.